blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+const rssTemplate = (
+  elements,
+  {
+    channelTitle,
+    channelDescription,
+    channelPath,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    imageTitle,
+    imageCaption,
+    imageCredits,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    includePromo,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      title: channelTitle || feedTitle,
```

Comment:
Since the alexa spec doesn't include this field I think you should make it optional like
...(channelTitle && {title: channelTitle})